File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        arr=[]
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if (nums[i]+nums[j]) == target:
                    arr.append(i)
                    arr.append(j)
        return arr             

File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> has;
        for(int i=0;i<nums.size();i++){
            int sum=target-nums[i];
            if(has.count(sum)){
                return {has[sum],i};
            }
            has[nums[i]]=i;
        }
        return {};
    }
};